packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCardPanel.styles.ts

```diff
@@ -11,6 +11,23 @@ export const getStyles = (): IAddCardPanelStyles => {
     },
     contentRoot: {
       margin: '0px 13px 20px 32px'
+    },
+    emptyCardStyles: {
+      position: 'absolute',
+      top: '50%',
+      left: '35%',
+      transform: 'translateY(-50%)',
+      margin: ' 0px auto',
+      height: '200px',
+      width: '200px'
+    },
+    emptyPanelMessageStyles: {
+      fontFamily: DefaultFontStyles.medium.fontFamily,
```

Comment:
Instead of pulling the font family from defaultFonStyles, could you access the theme and use theme.fonts.medium, like other code?